### PR TITLE
added fuel volume model

### PR DESCRIPTION
**Pull request by mjburton11**
Added a fuel volume constraint based on the surface area and the thickness to chord ratio.

### CODE REVIEW

**Comment by whoburg** [comment]
Let's clarify the purpose of this pull request -- the title says it's just fuel volume, but @1ozturkbe  told me yesterday that he wanted to add commits to the same branch.

Can someone please update the title and description so the purpose is clear?

File: 1682/GasPoweredHALE.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from gas_hale import GasPoweredHALE
+import gpkit
+import numpy as np
+gpkit.settings['latex_modelname'] = False
```
Comment (whoburg) [inline 1682/GasPoweredHALE.py]: no need to do `gpkit.settings['latex_modelname'] = False` in a `.py` file. This is done for ipython notebooks for clearer model latex printing when there is no model composition happening.

File: 1682/GasPoweredHALE.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from gas_hale import GasPoweredHALE
```
Comment (whoburg) [inline 1682/GasPoweredHALE.py]: What is the purpose of  `1682/GasPoweredHALE.py`?

**Comment by mjburton11** [comment]
Ok the purpose of my request was just the fuel volume.  Looks like @1ozturkbe added a bunch of stuff in addition to that.  @1ozturkbe you should probably rename it since you understand better what your commits were.  Maybe something like "added fuel volume, propellor, engine weight model".

File: 1682/GasPoweredHALE.py (hunk 2 above)
Comment (mjburton11) [inline 1682/GasPoweredHALE.py]: I was using this to get sensitivity plots.

**Comment by whoburg** [comment]
By the way, I'm making a bunch of comments because this is in a pull request, thus it triggers a review.

I'd like to clarify that we have two modes of operation:

1. Work on a branch but don't create a pull request. This is a useful way to share and collaborate on code that is evolving quickly and not ready for review.

2. Create a pull request to merge a branch into another branch. (the branch can of course be one that started out with no pull request as described above). This is a proposal to others, including myself, that the code be merged into the master branch, and triggers a thorough review/questioning/etc, which is a healthy process that improves the code, but does take some time.

Let me know if there are any questions -- hope that helps to clarify some of the options for using git to help with development.

**Comment by whoburg** [comment]
@mjburton11: if you want to get the fuel volume model on master here's the best way to do it, using cherry pick. Be sure to follow along as you execute the following commands and use your judgement -- I'm typing them without testing to make sure I included everything.

First, stash your work and `git checkout master`, and `git pull origin master`.  Make sure you have a clean master branch. Next,

```
git checkout -b fuel_volume
git cherry-pick 23cf1d6ca00a2f128e7669f5245dfd9e6c5221e8
git status
git push origin fuel_volumne
```
Then simply use the github web interface to make a PR from the fuel_volume branch -- it will only have the simple fuel volume commit.

File: 1682/GasPoweredHALE.py (hunk 2 above)
Comment (whoburg) [inline 1682/GasPoweredHALE.py]: great! Maybe move it into `gas_hale.py` and stick it in a method like
```python
def make_sensitivity_plots():
    # code goes here
```
You can then call `make_sensitivity_plots()` from the `if __name__ == "__main__"` section of gas_hale.py, or import `make_sensitivity_plots()` into an ipython notebook and use it there

Comment (mjburton11) [inline 1682/GasPoweredHALE.py]: Got it! Will do.

**Comment by mjburton11** [comment]
Moved to `gaspowered` branch.

**Comment by whoburg** [comment]
:+1:

**Pull request closed without merging**